fast/stages/1-resman/branch-nsec.tf

```diff
@@ -16,13 +16,25 @@
 
 # tfdoc:file:description Network security stage resources.
 
-# automation service account
+# TODO: remove in v35.0.0
 
 moved {
```

Comment:
Why aren't those in a `moved` file?

Reply:
because they are not a delta between versions